packages/open-payments/src/client/grant.ts

```diff
@@ -0,0 +1,58 @@
+import { HttpMethod } from 'openapi'
+import { ClientDeps } from '.'
+import {
+  getASPath,
+  InteractiveGrant,
+  InteractiveGrantRequest,
+  NonInteractiveGrant,
+  NonInteractiveGrantRequest
+} from '../types'
+import { post } from './requests'
+
+interface RequestGrantArgs<T> {
+  url: string
+  request: T
+}
+
+export interface GrantRoutes {
+  requestInteractiveGrant(
+    args: RequestGrantArgs<InteractiveGrantRequest>
+  ): Promise<InteractiveGrant>
+  requestNonInteractiveGrant(
+    args: RequestGrantArgs<NonInteractiveGrantRequest>
+  ): Promise<NonInteractiveGrant>
+}
+
+export const createGrantRoutes = (clientDeps: ClientDeps): GrantRoutes => {
```

Comment:
We'll also need the grant continuation route, in order to support the full interactive grant flow.
https://github.com/interledger/open-payments/blob/62c4b4a9875e3adaa21f89f597e88db43016fe0b/openapi/auth-server.yaml#L163-L171